src/spaces/test/spaces_test.cpp

```diff
@@ -1,50 +1,47 @@
 #include "src/spaces/spaces.hpp"
 
-#include <gtest/gtest.h>
+#include "boost/ut.hpp"
 
 // NOLINTBEGIN(readability-magic-numbers)
 
-namespace spaces {
+using vec = spaces::Vector<float, 3>;
+using point = spaces::Point<float, 3>;
 
-namespace test {}
-
-using vec = Vector<float, 3>;
-using point = Point<float, 3>;
-
-void spaces_vector()
+auto main() -> int
 {
-    constexpr vec zero{};
-    constexpr vec v1{1.0F, 0.0F, 0.0F};
-    constexpr vec v2{1.0F, 0.0F, -1.0F};
+    using namespace boost::ut;
 
-    static_assert(v1.get<0>() == 1.0F);
-    static_assert(v1.get<1>() == 0.0F);
-    static_assert(v1.get<2>() == 0.0F);
+    test("spaces vector") = [] {
+        constexpr vec zero{};
+        constexpr vec v1{1.0F, 0.0F, 0.0F};
+        constexpr vec v2{1.0F, 0.0F, -1.0F};
 
-    static_assert(v1 + zero == v1);
+        static_assert(v1.get<0>() == 1.0F);
```

Comment:
Take a look here:
https://github.com/boost-ext/ut#examples
```
// constant
constexpr auto compile_time_v = 42;
auto run_time_v = 99;
expect(constant<42_i == compile_time_v> and run_time_v == 99_i);
```
and here:
https://github.com/boost-ext/ut/issues/273

You can rewrite these 3 lines as:
```
expect(constant<v1.get<0>() == 1._f>);
expect(constant<v1.get<1>() == 0._f>);
expect(constant<v1.get<2>() == 0._f>);
```
so that they counted as tests by boost.ut.